python-threatexchange/threatexchange/cli/hash_cmd.py

```diff
@@ -103,7 +103,7 @@ def execute(self, api: ThreatExchangeAPI, dataset: Dataset) -> None:
 
         for inp in self.input_generator:
             hash_fn = lambda s, t: s.hash_from_file(t)
-            signal_types = file_hashers
+            signal_types: t.List[t.Any] = file_hashers
```

Comment:
nit: Can do with more typing at line 101.

Reply:
L101's type: `List[Type[<subclass of "SignalType" and "FileHasher">]]` I couldn't find a clean way to type that up there so wanted to minimize the amount of `t.Any` usage gets in the scope by adding the type hint later